products/sle/main.pm

```diff
@@ -131,6 +131,10 @@ if (is_sle('15+')) {
     # in the 'minimal' system role we can not execute many test modules
     set_var('INSTALLONLY', get_var('INSTALLONLY', check_var('SYSTEM_ROLE', 'minimal')));
 }
+# Always register against SCC if SLE 12 SP5
+elsif (is_sle('12-SP5+')) {
+    set_var('SCC_REGISTER', get_var('SCC_REGISTER', 'installation'));
```

Comment:
can you provide a validation run also for staging DVDs? I have a suspicion...

Reply:
As the staging tests don't even reach this module I won't be able to do a verification run.
 I just updated this PR so that it will not change anything in case of staging.